src/dispatcher/mod.rs

```diff
@@ -159,6 +167,20 @@ impl Dispatcher {
                     name
                 )));
             }
+            // check if service is not yet up but started
+            if let Some(_) = service.url {
```

Comment:
Again this affect all services up or starting up

Reply:
right. i dont want to be able to start another duplicate service if its starting because it will fail anyways. i would want to do a 'up' only if the service is down